tasks/f35/f35_postgresql.yml

```diff
@@ -14,8 +14,9 @@
 
 - name: "Initialize PostgreSQL database"
   become: true
-  command:
-    cmd: postgresql-setup --initdb --unit postgresql
+  ansible.builtin.command:
+    # cmd: postgresql-setup --initdb --unit postgresql
+    cmd: su - postgres -c "PGDATA=/var/lib/pgsql/data initdb"
```

Comment:
I have no idea why this breaks; postgresql-setup can't connect at all?

Reply:
PostgreSQL provides so many ways to initialize it, and they all break or change from env to env, which is frustrating.

After testing a bit, I've landed on the following:

```
- name: "Initialize PostgreSQL database"
  become: true
  become_user: postgres
  ansible.builtin.command:
    cmd: pg_ctl initdb -D /var/lib/pgsql/data
    creates: "/var/lib/pgsql/data/PG_VERSION"
    warn: false
  tags:
    - postgresql
    - candlepin
```

It's pretty close to what you have already, but uses what appears to be the new(?) centralized entry point for pg administrative tasks, moves the environment variable to an explicit parameter, and moves the user switching up to the Ansible task config.